nav2_collision_monitor/src/polygon.cpp

```diff
@@ -372,6 +363,28 @@ bool Polygon::getCommonParameters(std::string & polygon_pub_topic)
         node, polygon_name_ + ".polygon_pub_topic", rclcpp::ParameterValue(polygon_name_));
       polygon_pub_topic = node->get_parameter(polygon_name_ + ".polygon_pub_topic").as_string();
     }
+
+    if (action_type_ == STOP || action_type_ == SLOWDOWN || action_type_ == LIMIT ||
```

Comment:
`if (action_type_ != APPROACH) {} else {}` is more concise

Reply:
I kept the test as it was before, but you're right, I changed it.